Tensile/KernelWriterAssembly.py

```diff
@@ -1814,6 +1833,164 @@ def functionSignature(self, kernel ):
     kStr += self.comment1("DirectToLdsB=%s" % kernel["DirectToLdsB"])
     kStr += self.comment1("UseSgprForGRO=%s" % kernel["UseSgprForGRO"])
 
+    if kernel["ProblemType"]["DataType"].isHalf():
+      if kernel["ProblemType"]["HighPrecisionAccumulate"]:
+        srcSize = "2"
+        srcAlign = "2"
+        srcValueType = "F16"
+        dstSize = "2"
+        dstAlign = "2"
+        dstValueType = "F16"
+        cptSize = "4"
+        cptAlign = "4"
+        cptValueType = "F32"
+      else:
+        srcSize = "2"
+        srcAlign = "2"
+        srcValueType = "F16"
+        dstSize = "2"
+        dstAlign = "2"
+        dstValueType = "F16"
+        cptSize = "2"
+        cptAlign = "2"
+        cptValueType = "F16"
+    
+    elif kernel["ProblemType"]["DataType"].isInt8x4():
```

Comment:
is this a vector of char or array of char in the function declaration?

Reply:
kernel is a Python dict,